src/NodeManipulator/ClassConstManipulator.php

```diff
@@ -25,9 +26,9 @@ public function __construct(
 
     public function hasClassConstFetch(ClassConst $classConst, ClassReflection $classReflection): bool
     {
-        $class = $this->betterNodeFinder->findParentType($classConst, Class_::class);
-        if (! $class instanceof Class_) {
-            return false;
+        $class = $this->betterNodeFinder->findParentByTypes($classConst, [Class_::class, Enum_::class]);
+        if (! $class instanceof \PhpParser\Node\Stmt\ClassLike) {
```

Comment:
`ClassLike` already in use statement, can just use as is:

```suggestion
        if (! $class instanceof ClassLike) {
```